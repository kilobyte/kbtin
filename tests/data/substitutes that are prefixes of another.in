#sub foo FOO
#sub foobar FOOBAR
#run a {sh -c 'echo xfoox;echo xfoobarx'}
