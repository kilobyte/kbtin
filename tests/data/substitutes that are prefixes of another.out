xFOOx
xFOOBARx

#SESSION 'a' DIED.
#THERE'S NO ACTIVE SESSION NOW.
